plugin.go

```diff
@@ -145,10 +150,14 @@ func planCommand(variables map[string]string, secrets map[string]string, paralle
 	)
 }
 
-func applyCommand(parallelism int) *exec.Cmd {
+func applyCommand(parallelism int, targets []string) *exec.Cmd {
 	args := []string{
 		"apply",
 	}
+	for _, v := range targets {
+		args = append(args, "--target")
```

Comment:
This line can be combined with the next to be args = append(args, "--target", fmt.Sprintf("%s", v))